src/client/components/MealList.jsx

```diff
@@ -0,0 +1,64 @@
+import "./MealList.css";
+import React, { useState, useEffect } from "react";
+import Meal from "./Meal";
+import { Container, Card, Row, Col } from "react-bootstrap";
+import { Link, useLocation } from "react-router-dom";
+
+function MealList() {
+  const [meals, setMeals] = useState([]);
+  const location = useLocation();
+
+  useEffect(() => {
+    fetchMeal();
+  }, []);
+
+  const fetchMeal = async () => {
+    try {
+      const response = await fetch("http://localhost:4000/api/meals");
+      const data = await response.json();
+      setMeals(data);
+      console.log(data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const RenderMeals = () => {
+    let mealsToRender;
+
+    if (location.pathname === "/") {
```

Comment:
Usually you would use the React Router for things like this, but I guess it was covered in a later class.
If you have the time, I recommend trying to restructure your code to use React Router to find out how many meals to render.